src/components/WalletTransactionDetails.vue

```diff
@@ -141,10 +211,43 @@ export default class WalletTransactionDetails extends Mixins(TranslationMixin, N
     return this.selectedTransaction.symbol2 || '';
   }
 
-  get transactionDate(): string {
+  get transactionFromDate(): Nullable<string> {
+    if (!this.selectedTransaction.startTime) return null;
+
     return this.formatDate(this.selectedTransaction.startTime as number);
   }
 
+  // ETH BRIDGE transaction
+  get transactionToDate(): Nullable<string> {
+    if (!this.selectedTransaction.endTime) return null;
+
+    return this.formatDate(this.selectedTransaction.endTime as number);
+  }
+
+  get transactionFromAddress(): Nullable<string> {
+    return this.getTransactionAddress(this.isSoraTx, true);
+  }
+
+  get transactionToAddress(): Nullable<string> {
+    return this.getTransactionAddress(this.isSoraTx, false);
+  }
+
+  get transactionFromHash(): { value: Nullable<string>; hash: Nullable<string>; translation: string; type: HashType } {
+    const { value, type } = this.getTransactionId(this.isSoraTx);
+    const hash = this.getTransactionHash(this.isSoraTx);
+    const translation = this.getTransactionTranslation(this.isSoraTx);
+
+    return { value, hash, translation, type };
+  }
+
+  get transactionToHash(): { value: Nullable<string>; hash: Nullable<string>; translation: string; type: HashType } {
+    const { value, type } = this.getTransactionId(!this.isSoraTx);
```

Comment:
It seems that transactionFromHash and transactionToHash are almost the same (there are only one difference this.isSoraTx instead of !this.isSoraTx. Do you think this is a good idea to have a method for them and use it in both places?